### ISSUE TITLE
Support for latest Vite version

### ISSUE THREAD

**Issue by hiepvq** [kucrut/vite-for-wp#81]
Cannot install, conflict with another plugin
=====
npm add -D vite @kucrut/vite-for-wp

```js
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: smfw@1.0.0
npm ERR! Found: vite@5.1.4
npm ERR! node_modules/vite
npm ERR!   dev vite@"^5.1.4" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer vite@"^4.4.11" from @kucrut/vite-for-wp@0.8.0
npm ERR! node_modules/@kucrut/vite-for-wp
npm ERR!   dev @kucrut/vite-for-wp@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

**Comment by j7-dev** [comment]
work with vite@5.2.7

**Comment by kucrut** [comment]
This should be fixed by #92. I'll release the new version soon.
